lib/resty/kafka/client.lua

```diff
@@ -188,6 +189,8 @@ local function _fetch_metadata(self, new_topic)
             -- we have been referred to. This injects the SASL auth in.
             for _, b in pairs(brokers) do
                 b.sasl_config = sasl_config
+                local h = b.host
+                b.host = sc.resolver and sc.resolver(h) or h
```

Comment:
```suggestion
                b.host = sc.resolver and sc.resolver(b.host) or b.host
```
this style is better.